app/assets/javascripts/routes/reviews_index.js

```diff
@@ -0,0 +1,5 @@
+App.ReviewsIndexRoute = App.AuthenticatedRoute.extend({
+  model: function() {
+    return this.modelFor('course').get('reviews');
```

Comment:
This is a good point.  It brings up the "what is the right behavior when a user navigates away from a field without saving it?"

We could:

1.  destroy the review when the user leaves without saving.
2.  include the half finished review in everything
3.  leave the unfinished review in existence but exclude it from the list
  *  This will allow them to finish it later when they nav back to new review

I vote for 1